### ISSUE TITLE
Linker Error while building

### ISSUE THREAD

**Issue by jonaswgit** [quadule/knobby#4]
Hello,
first of all thanks for the great Idea of this Project and sharing it!

Unfortunately I cannot build the Code. I'm running Ubuntu 18.04, and `platformio run` fails with a linker error:
```
Building in release mode
Linking .pio/build/knobby/firmware.elf
/home/XXX/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: .pio/build/knobby/src/main.o:(.literal._ZNSt17_Function_handlerIFvP21AsyncWebServerRequestEZ5setupvEUlS1_E3_E9_M_invokeERKSt9_Any_dataOS1_+0x20): undefined reference to `_binary_index_html_start'
/home/XXX/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: .pio/build/knobby/src/main.o:(.literal._Z5setupv+0x20): undefined reference to `_binary_GillSans24_vlw_start'
/home/XXX/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: .pio/build/knobby/src/main.o:(.literal._Z5setupv+0x24): undefined reference to `_binary_icomoon24_vlw_start'
/home/XXX/.platformio/packages/toolchain-xtensa32/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: .pio/build/knobby/src/main.o:(.literal._Z5setupv+0x28): undefined reference to `_binary_icomoon31_vlw_start'
collect2: error: ld returned 1 exit status
```
Do you have an Idea what could be the cause?

**Comment by quadule** [comment]
Ah, looks like PlatformIO released a platform update that required some additional configuration — should be fixed in 4732d67e8e32af5acc06981c392d7eeffa8ddf15. Thanks for reporting this!

**Comment by jonaswgit** [comment]
Yes, its fixed. Thank you!
